graph.go

```diff
@@ -106,7 +107,7 @@ func (c *GraphCmd) Execute(args []string) error {
 				}
 
 				oldSrcDir := filepath.Join(dir, "src")
-				newGOPATH := filepath.Join("/tmp/gopath-" + strconv.Itoa(i) + "-" + filepath.Base(dir))
+				newGOPATH := path.Join(os.TempDir(), "gopath-"+strconv.Itoa(i)+"-"+filepath.Base(dir))
```

Comment:
Yep, I did this in a few places while fixing things on auto-pilot it looks like. Should definitely be `filepath.Join`.